process/block/metablock.go

```diff
@@ -86,6 +86,7 @@ func NewMetaProcessor(arguments ArgMetaProcessor) (*metaProcessor, error) {
 		return nil, process.ErrNilEpochStartSystemSCProcessor
 	}
 
+	pruningDelay := uint32(arguments.Config.StateTriesConfig.PeerStatePruningQueueSize * pruningDelayMultiplier)
```

Comment:
could add a check for `arguments.Config.StateTriesConfig.PeerStatePruningQueueSize` . If the config is missing, then it will be 0 and `pruningDelay` will be `0`. 
this means that `if bp.blocksSinceLastRestart <= bp.pruningDelay {` will be 99.99% of times false, and the pruning will be active

Reply:
Done.